contrib/golang/filters/http/test/golang_integration_test.cc

```diff
@@ -1093,9 +1158,9 @@ TEST_P(GolangIntegrationTest, Async_DataBuffer_DecodeHeader) {
 TEST_P(GolangIntegrationTest, DataBuffer_DecodeData) { testBasic("/test?databuffer=decode-data"); }
 
 // buffer all data in decode data phase with async mode.
-TEST_P(GolangIntegrationTest, Async_DataBuffer_DecodeData) {
-  testBasic("/test?async=1&databuffer=decode-data");
-}
+// TEST_P(GolangIntegrationTest, Async_DataBuffer_DecodeData) {
```

Comment:
by accident?

Reply:
Uff yes sorry I though I did not commit this....